Makefile

```diff
@@ -68,6 +68,8 @@ docker-image:
 		--build-arg git_sha="$(GIT_SHA)" \
 		--build-arg version="$(VERSION)" \
 		.
+
+.PHONY: build-ttl.sh
```

Comment:
What does mean ttl.sg?
Why do we need to do this build
Is possible to add a comment into the target for we have doc (what/why/when) and we are able to check with make --help?

Reply:
Done